autogpt_platform/backend/backend/blocks/taskmarket/blocks.py

```
@@ -0,0 +1,299 @@
+    class Output(BlockSchemaOutput):
+        task_id: str = SchemaField(description="Created TaskMarket task ID")
+        task_url: str = SchemaField(description="Canonical TaskMarket task link")
+        live_status: dict[str, Any] = SchemaField(
+            description="Live task state read back after creation"
+        )
+
+    def __init__(self) -> None:
+        test_preview = TaskMarketTaskPreview.build(
+            description="Write an accessibility report",
+            deliverables=["report.md"],
+            reward_usdc=Decimal("2"),
```

Ruff (0.16.1)

[error] Verbose expression in `Decimal` constructor

Replace with `2`

(FURB157)